Vulkan-Projects.adoc

```diff
@@ -0,0 +1,162 @@
+// Copyright 2017-2021 The Khronos Group Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+# Map of Vulkan project GitHub repositories
+
+Vulkan is a big place and there's a lot out there that is useful! In no particular order...
+
+## Khronos/Vulkan® Specification
+Vulkan® Specification - formal documentation of the Vulkan API. This includes the main API Specification, the reference (man) pages, the XML API Registry, and related tools and scripts.
+
```

Comment:
Might say something like "the published [Vulkan specifications and reference pages](https://www.khronos.org/registry/vulkan/#apispecs), as well as the published [header files](https://github.com/KhronosGroup/Vulkan-Headers), are generated from the sources in this repository."

Reply:
Or even the longer descriptions in index.php. I'm not sure how to best reduce replication in these two documents but do want to make sure we don't lose something useful in replacing the material under https://www.khronos.org/registry/vulkan/#repos with this page. Also VK-GL-CTS repository is in index.php but I think not here, yet - should it be?

Reply:
I added a "see also" statement at the top with link to this repo list...not quite sure if we want to duplicate info here.  Perhaps we should consider just pointing folks at the #repos link instead and not even have this file??  Main difference is the SPIRV and compiler stuff - and MoltenVK...perhaps add those to #repos list?

Reply:
Whichever way you think best. Regarding the Docker image, there's no point in providing a link - you specify the image to the docker app exactly as I suggested, and it knows how to look up on dockerhub.

Reply:
Lets leave it here for now...this stuff doesn't change very often, so shouldn't be a problem to maintain.  I fixed compiler item to point to dockerhub